pkgs/development/python-modules/sphinx-argparse/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, CommonMark
+, sphinx
+}:
+
+buildPythonPackage rec {
+  pname = "sphinx-argparse";
+  version = "0.2.5";
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "05wc8f5hb3jsg2vh2jf7jsyan8d4i09ifrz2c8fp6f7x1zw9iav0";
+  };
+
+  propagatedBuildInputs = [
+    CommonMark
```

Comment:
As this is an optional dependency, what's your argument for including it by default?

Reply:
None. Removed.